src/services/content/docs/index.js

```diff
@@ -0,0 +1,77 @@
+module.exports = {
+	contentService: {
+		resourcesService: {
+			description: "A Service for content",
+			get: {
+				parameters: [
+					{
+						description: "id of the course",
```

Comment:
I don't think it's the id of a course, but rather the id of the resource you are trying to request.

Reply:
yeah just checked... has absolutely nothing to do with the courseId